src/Libraries/Inscriptions/Envelope.ts

```diff
@@ -43,8 +43,9 @@ export class Envelope {
     readonly txid: string,
     readonly data: EnvelopeData[],
     readonly oip?: object,
+    readonly index: number = 0,
```

Comment:
I think optional params have to be the last param? so the `oip?` param has to be the last one. The `index` can be moved above `oip?`

Reply:
can can